pages/calculator/[slug].js

```diff
@@ -195,26 +223,50 @@ export default function CalculatorSlugRoute({ page, calculatorConfig }) {
                 >
                   {calculatorConfig.feedback.linkText}
                 </Button>
-                {/* check another conviction */}
-                <Link
-                  sx={{ textAlign: 'center' }}
-                  href={
+
+                <Box sx={{
+                  display: 'flex', flexDirection: 'column', gap: 1.5, marginLeft: matchesXS ? 0 : 7,
```

Comment:
Have you tried minimizing more CSS? If you take a look at the entire file, most of the CSS is concentrated to where you have added the share code. I just deleted the entire `sx` here, and the resulting page looks identical.

I actually just went an deleted 100% of the added CSS in this file, and the page looks almost the same. If slight things ever need to change, this component will be hard to maintain with all of the extra but not needed styling. I would try maybe commenting out all of your added styling, and then starting from there, see which pieces are truly necessary. I totally see where some will be needed, I am not saying go to nothing, but we are using the MUI components that already have base styles, so that we can reduce the CSS we have to maintain.

Reply:
So Eleni I am removing some alot of the styles but it does look a little different so I am not sure what I should be prioritizing  here: the vision of the designer or our accessibility with the code. I'll push the changes and maybe you can see if the changes are too much or not. Also if you haven't, please check mobile view since it does affect that.